pkg/fanal/analyzer/language/java/sbt/lockfile_test.go

```diff
@@ -0,0 +1,96 @@
+package sbt
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+)
+
+func Test_sbtDependencyLockAnalyzer(t *testing.T) {
+	tests := []struct {
+		name string
+		dir  string
+		want *analyzer.AnalysisResult
+	}{
+		{
+			name: "empty lockfile",
+			dir:  "testdata/empty",
+			want: &analyzer.AnalysisResult{
+				Applications: []types.Application{
+					{
+						Type:     types.Sbt,
+						FilePath: "build.sbt.lock",
+						Packages: types.Packages{},
+					},
+				},
+			},
+		},
```

Comment:
nit:
we try to use tests with results first (`v1 lockfile` -> `empty lockfile` in your case).